app/uk/gov/hmrc/timetopay/arrangement/config/JurisdictionCheckerConfig.scala

```diff
@@ -17,21 +17,19 @@
 package uk.gov.hmrc.timetopay.arrangement.config
 
 import javax.inject.Inject
-
 import play.api.Configuration
 
-
-case class JurisdictionCheckerConfig @Inject()(scottishPrefix:String , welshPrefix:String)
+case class JurisdictionCheckerConfig @Inject() (scottishPrefix: String, welshPrefix: String)
```

Comment:
Remove @inject, looks like it's not used

Reply:
done